proxima/code/game/items/toys.dm

```diff
@@ -0,0 +1,31 @@
+/obj/item/toy/figure/plush/skrell
+	icon = 'proxima/icons/obj/toys.dmi'
+	name = "blue plush skrell"
+	desc = "A soft plush toy is so nice to hug her, but on a piece of paper next to it it says: NOT TO USE AS A TOY FOR JOY"
```

Comment:
`her` -> `it`, toys have only neutral gender since they are soulless, as the person writing this quote.
`next to it it says` -> `next to it says` since piece of paper is the noun of objection. Same applies to every other descriptions